cli/tests/testdata/test/steps/invalid_usage.ts

```diff
@@ -11,7 +11,7 @@ Deno.test("capturing", async (t) => {
 
 Deno.test("top level missing await", (t) => {
   t.step("step", () => {
-    return new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
Why was this changed?

Reply:
It's an async op leak that's dependent on the following test having sanitizers off. (Some approach I tried while WIP exposed that but it's back to normal)